src/meta/src/controller/streaming_job.rs

```diff
@@ -497,6 +476,12 @@ impl CatalogController {
                     }
                     .update(&txn)
                     .await?;
+
+                    if is_materialized_view {
```

Comment:
Shall we also expose internal tables for other streaming jobs (like sink)?

Reply:
Can sink DDL be blocking now?

Reply:
Sink DDL is already blocking by default during the backfill phase I think 🤔

Reply:
> Shall we also expose internal tables for other streaming jobs (like sink)?
https://github.com/risingwavelabs/risingwave/issues/20522